packages/xo-web/src/common/intl/messages.js

```diff
@@ -77,6 +77,7 @@ const messages = {
   withMemory: 'With memory',
   offline: 'Offline',
   noLicenseAvailable: 'No license available',
+  emailPlaceholderExample: 'Enter an email address, e.g: it@enterprise.com',
```

Comment:
I think 'Email address' is enough.

Reply:
IMHO, adding an example it always a good think. Please don't forget to assign the second reviewer.

Reply:
What about the label properly describing what's expected (`Report recipients email addresses`) and then the placeholder giving an additional hint (`it@company.net`)?

Reply:
It's a long title. What's wrong with my suggestion?

Reply:
`Email address, e.g: it@enterprise.com`?